lib/registration.pm

```diff
@@ -431,7 +431,6 @@ sub fill_in_registration_data {
                 assert_screen 'yast-scc-emptypkg';
                 send_key 'alt-a';
             }
-            accept_addons_license('ha') if (check_var('SLE_PRODUCT', 'sles4sap'));
```

Comment:
are you sure ? this isnt produduct bug ?

Reply:
not on SLE12SP3, there is no HA at all, but on SLE 15 maybe, there are multiple modules and not single license shown
[SLE12SP3](http://10.100.12.155/tests/5589#step/scc_registration/11)
[SLE15](https://openqa.suse.de/tests/1772887#step/scc_registration/9)

Reply:
ah i just found the licenses https://openqa.suse.de/tests/1772887#step/accept_license/1